### PR TITLE
Update Guides

### PR DESCRIPTION
**Pull request by yash-sharma1**
## Pre-flight Checklist

Please complete this checklist before requesting a review:

- [x] I have performed a self-review of my code / documentation
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: docs/guides/troubleshoot-cors-errors.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Troubleshooting CORS Errors
+sidebar_position: 5
+---
+
+import Link from "@docusaurus/Link";
```
Comment (pyr0gan) [inline docs/guides/troubleshoot-cors-errors.mdx:6]: This component isn't used here, the links here are all markdown style. 
As in the network guide, the syntax for a Docusaurus Link is

```jsx
<Link to="/someroute">Label Text</Link>
```

@yash-sharma1, we might convert the link style to use the component. Or remove the import if we aren't using it.

File: docs/guides/troubleshoot-cors-errors.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+2. **Use a serverless function** - Using a serverless function is another more effective way to handle the CORS errors and proxy our requests and here in this we build our own functions or micro-infrastructure to call a web service and interact using APIs. Azure, AWS & GCP are most popular for running serverless functions.
+
+   Sharing some examples of serverless functions built and shared by our community members.
+
+   - Azure Function for CORS proxy by [Florian Uhde](https://github.com/floAr) - Check the [Gist HERE](https://gist.github.com/floAr/95db6ed96b1b0cf70a0513698a3979f7)
+   - AWS Lambda Function for CORS proxy by [Austin Woetze](https://github.com/AustinWoetzel)- Check the [Gist HERE](https://gist.github.com/AustinWoetzel/da808287f35effd2419ef44ed8593054)
+   - Template if you want to host your own nginx server - [DataHub CORS Development Proxy](https://github.com/figment-networks/datahub-cors-dev-proxy)
+
+_References -_
```
Comment (pyr0gan) [inline docs/guides/troubleshoot-cors-errors.mdx:33]: Might be better as a heading? Try H3!

**Review by pyr0gan — COMMENTED** [review]
@yash-sharma1 request a review from @CharlesJonesFigment also 💪 

 Maybe best to strategize on the link formatting with @paulpomerleau. There are benefits to using the component, although there's nothing wring with using the markdown style either.

File: docs/guides/troubleshoot-cors-errors.mdx (hunk 1 above)
Comment (paulpomerleau) [reply docs/guides/troubleshoot-cors-errors.mdx:6]: let's give the component a shot

File: docs/guides/troubleshoot-cors-errors.mdx (hunk 2 above)
Comment (paulpomerleau) [reply docs/guides/troubleshoot-cors-errors.mdx:33]: h4 it is

File: docs/guides/troubleshoot-cors-errors.mdx (hunk 1 above)
Comment (yash-sharma1) [reply docs/guides/troubleshoot-cors-errors.mdx:6]: Okay!

File: docs/guides/troubleshoot-cors-errors.mdx (hunk 2 above)
Comment (yash-sharma1) [reply docs/guides/troubleshoot-cors-errors.mdx]: Resolved

File: docs/guides/troubleshoot-cors-errors.mdx (hunk 1 above)
Comment (yash-sharma1) [reply docs/guides/troubleshoot-cors-errors.mdx:6]: Resolved

**Pull request merged**